codegen-server/src/main/kotlin/software/amazon/smithy/rust/codegen/server/smithy/protocols/ServerHttpProtocolGenerator.kt

```diff
@@ -155,26 +153,6 @@ private class ServerHttpProtocolImplGenerator(
         operationShape: OperationShape
     ) {
         val errorSymbol = operationShape.errorSymbol(symbolProvider)
-        val successCode = httpBindingResolver.httpTrait(operationShape).code
```

Comment:
The server doesn't need to parse responses.

Reply:
Yep, this can go. It was here only to be able to build the protocol tests, which we are changing. Good to see this going :)